packages/generate-persisted-query-manifest/src/index.ts

```diff
@@ -401,6 +403,13 @@ function formatErrorMessage(error: Error) {
   return `${error.name}: ${error.message}`;
 }
 
+function getClientVersion() {
```

Comment:
Unfortunately we don't export `version` from `@apollo/client` in v3 like we do in v4 so we have to get the version this way.